### ISSUE TITLE
It is not obvious where arrow keys will navigate to (Column mode)

### ISSUE THREAD

**Issue by eljefuri** [elementary/files#691]
Lets say that I have highlighted one of the default folders in the sidebar, and then I press the right arrow key → with the intention of entering into keyboard navigation in that folder. There is no visual indication that I have done this, but I can then press the down arrow ↓ and it is then apparent where I am. The same is true within columns. Pressing the right arrow into a folder does not give any visual indication that my arrow key navigation _is now constrained to that subfolder_. In macOS Finder, there is always an indicator. This issue is probably related to #690

Thank you for reading and for your consideration of the issue.

**Comment by jeremypw** [comment]
It sounds like you prefer keyboard operation rather than mouse and it is true that less work is done to optimise keyboard operation than mouse operation.  As you say this issue is related to the whole matter of how to visually indicate (and distinguish) keyboard focus,  cursor position, hovered items, selected items etc.

**Comment by eljefuri** [comment]
Yes, it sounds like it. Where can I go to discuss this sort of design, or would that happen in this thread?

**Comment by jeremypw** [comment]
I am not sure what you meant by *constrained to that sub-folder*.  Pressing left should navigate to the previous column and pressing right (on a folder) should open another subfolder.

When the keyboard focus is in a column where no item is selected, the cursor position is indicated by highlighting the icon (may not be very obvious).  At least, this happens in the master version.

**Comment by jeremypw** [comment]
Actually, this may be more related to #552.  See also #473.

If you feel there is a distinct issue here please let me know, otherwise I'll mark this as duplicate and further discussion can take place in the original report.

You can have general discussions on design on the Slack UX Channel, if you have access, or one of the elementary social forums, I guess.  If there is a clear pathway/consensus then a project can be created.

**Comment by eljefuri** [comment]
Thanks for that @jeremypw I think they are indeed both related to this but I am not sure if one or the other is more relevant, or if I am pointing out something more specifically nuanced regarding keyboard navigation.

Regarding the discussion: Slack is really not a good medium to have a fixed, lengthy discussion which is easily able to be referenced or commented upon over time. We need something like a forum, which is what GitHub issues helpfully provides -- although I get the fine line. The Elementary social forums are also not ideal because apparently the developers do not often comment on them. GitHub is really the best place for this that I have seen, unless there is an active Discord or equivalent forum somewhere which the developers also frequent?

**Comment by jeremypw** [comment]
I would not be averse to creating a project "Make Files more suitable for power users", but even projects do not have anywhere for general discussion.  They just link together a number of specific issues (which could be wishlist type) where specific discussions could take place.  We still receive comments on closed issues (at least, I do) so discussion can continue there, but it is not discoverable by others outside elementary.

**Comment by eljefuri** [comment]
_@jeremypw This veers quickly off-topic, but I want to suggest in the case of open discussions that maybe a special GitHub tag is constructed for this purpose: "discussion" or "broad topic" or something. The idea being that things tagged with that come with certain rules, such as a limited period of time in which they can be an "open" issue (say, 1 month maximum). This would be a compromise for both sides that at least I find acceptable for now._

**Comment by eljefuri** [comment]
@jeremypw I think the tag "Opinion" would be perfect if it had a 1-month discussion shelf life or so, to at least give it exposure (and after that conversation could still be continued in the closed issue).

**Comment by jeremypw** [comment]
If desired, a Discussion could be opened around this and related issues of file item styling.

**Comment by danirabbit** [comment]
I think this is also a duplicate of #473. Files is using a custom focus style that is pretty hard to see in column view
